utils/nba_api_client.py

```
@@ -0,0 +1,175 @@
+    def _read_cache(self, name, params):
+        path = self._cache_path(name, params)
+        if not os.path.exists(path):
+            return None
+        with open(path, "r", encoding="utf-8") as f:
+            cached = json.load(f)
+        timestamp = cached.get("timestamp", 0)
+        if time.time() - timestamp > self.cache_ttl:
+            return None
+        return self._deserialize(cached.get("payload"))
+
+    def _write_cache(self, name, params, payload):
```

Ruff (0.14.10)

Missing return type annotation for private function `_write_cache`

Add return type annotation: `None`

(ANN202)